packages/frontend/src/components/DashboardTiles/EmptyStateNoTiles/index.tsx

```diff
@@ -88,6 +92,7 @@ const EmptyStateNoTiles: FC<SavedChartsAvailableProps> = ({
                             <SavedChartsAvailable
                                 onAddTiles={onAddTiles}
                                 isEditMode={isEditMode}
+                                dashboardName={dashboardName}
```

Comment:
could we use the dashboard provider/context to avoid all the prop drilling?

Reply:
done!